docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -0,0 +1,70 @@
+# Signalen and the Standard for Public Code
+
+## [Code in the open](https://standard.publiccode.net/criteria/code-in-the-open.html)
+
+* Current backlog and bug-tracking is not public. Some issues referenced in code and commits are not easy to locate, like "SIG-1525". Transition to public bug-tracker?
+* Can the classification system be completely open sourced? How can we address risks and possibilities regarding (of theoretical) leaking of information. Maybe add anonymized MOR datasets or redo the classification algorithm with public available MOR data sets. Can a basic Common Ground version be made?
+* Citizens are not authenticated. Data is not cleaned up for sharing (privacy purposes). https://github.com/Amsterdam/signals/blob/master/docs/api-design.md#publiek-en-ongeauthenticeerd
+
+## [Bundle Policy And Source Code](https://standard.publiccode.net/criteria/bundle-policy-and-code.html)
+
+* Making the state machine configurable would enable Signalen to adapt to policy changes within Amsterdam and also enable replicators to customise Signalen for their workflows.
```

Comment:
We will change the phrasing slightly, to refer to the more general problem of hardcoded business rules (it is not just the state machine of Signals).

Reply:
Changed the phrasing a bit so that it also includes the business ruling that is needed